### ISSUE TITLE
No apps in the output

### ISSUE THREAD

**Issue by jettro** [bwesterb/blame-bird#4]
Hi running the latest Sierra release 10.12.2. I get nothing but this:

Accounted for: 0MB.  Still unaccounted: 7291MB

I guess no known applications? Could it be something else?

**Comment by bwesterb** [comment]
I reverse engineered part of the bird index: on my system I know for most files what they do, but I couldn't figure out where some came from.  That's why I have the "Still unaccounted".  Apparently, in your case, the stray files are all in the index I haven't figured out yet.

**Comment by jettro** [comment]
Fair enough, will try to find some time to see if I can extend the script to understand more applications.

**Comment by bwesterb** [comment]
It could also be that the format of the index database has changed.  I'm abroad for a week, so I won't have time to check.

**Comment by JohnSmith843** [comment]
Hi, I have the same issue. Here is what I got back

```
Accounted for: 0MB.  Still unaccounted: 35125MB
MacBook-8318:~ nicolasfafchamps$ 
```

Any help, would be much appreciated. I really wonder were those 35000MB come from.

**Comment by bwesterb** [comment]
My mac is on High Sierra now, where they changed the way bird works — so I'm unable to help you out.

**Comment by JohnSmith843** [comment]
I am on the latest release of Sierra (not high sierra). There is still no solution ?

**Comment by bwesterb** [comment]
As I said, I reverse engineered parts of the way bird works.  Bird works differently on High Sierra than it does on Sierra.  So, no, without having access to Sierra, I can't help you.  You can try to reverse engineer it yourself, off course.

**Comment by JohnSmith843** [comment]
Ok I understand. Thanks for your help anyway. 

I took me half an hour to figure out how to run a python script, I am very new to this. So it is way outside my abailities to reverse engineer anything.
